spec/index.html

```diff
@@ -778,7 +778,7 @@ <h2>Literals and datatypes</h2>
     The <a>value space</a> of <code>rdf:langString</code> is the set of all pairs of a string with a language tag.
     The semantics of literals with this as their type are given below.</p>
 
-  <p>RDF literal syntax allows any IRI to be used in a typed literal,
+  <p>RDF literal syntax allows any IRI to be used in a literal,
```

Comment:
Other than for the language-tagged string syntax, which is a literal that does not allow an IRI to be specified. Perhaps just add a parenthetical.